src/App/Components/NotFound/NotFound.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 const NotFound = () => (
-  <React.Fragment>
-    <h1>Authors Heaven</h1>
-    <h2>404</h2>
-    <p>The page you are searching for was not found</p>
-  </React.Fragment>
+  <div className="notFound">
```

Comment:
Why there two not found pages in your PR? @MCFrank16

Reply:
One is for an article and the other is for the page, but it can still be changed no worries

Reply:
USE ONE PLEASE...MCFrank16